From 42d800f377dfa4e7338c516d286aef0b76bb1f72 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Thu, 13 Oct 2011 18:58:57 -0400
Subject: [PATCH] Drop an unneeded include

GLib recently became less forgiving about single includes
like this.
---
 client/gvfsfusedaemon.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/client/gvfsfusedaemon.c b/client/gvfsfusedaemon.c
index b41e83b..a4ba32c 100644
--- a/client/gvfsfusedaemon.c
+++ b/client/gvfsfusedaemon.c
@@ -41,8 +41,6 @@
 #include <glib/gprintf.h>
 #include <gio/gio.h>
 
-#include <glib/gurifuncs.h>
-
 /* stuff from common/ */
 #include <gdaemonmount.h>
 #include <gvfsdaemonprotocol.h>
-- 
1.7.8.4

